tailwind.config.js

```diff
@@ -1,8 +1,45 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
+  darkMode: 'class',
```

Comment:
Ideally this is the only part that’s necessary outside of the class name additions themselves.